src/edu/cs505/g5/RasberriPiState.java

```diff
@@ -0,0 +1,23 @@
+package MagicMirror2;

```

Comment:
You have the package as MagicMirror2, but your code is in the directory edu.cs505.g5 thus compile error

Reply:
modified